hs_core/tasks.py

```diff
@@ -90,7 +89,7 @@ def manage_task_nightly():
             if response.status_code == status.HTTP_200_OK:
                 # retry of metadata deposition succeeds, change resource flag from failure
                 # to pending
-                res.doi = get_resource_doi(act_doi, 'pending')
+                res.doi = act_doi
```

Comment:
ok, so the doi was built twice, once correctly to act_doi, then the doi was passed in as a resource id.

Reply:
It's not clear to me how the edit published resource feature caused an issue here.